src/Node/EmbedNode.php

```diff
@@ -49,7 +49,11 @@ protected function addGetTemplate(Compiler $compiler, string $template = ''): vo
         if ($this->getAttribute('ignore_missing')) {
             $compiler
                 ->raw(";\n")
+                ->write(sprintf("if (\$%s) {\n", $template))
```

Comment:
This is always `true` as `$template` is never empty when `ignore_missing` is set.

Reply:
Ok, reverted.